### ISSUE TITLE
Change presentation of private messages

### ISSUE THREAD

**Issue by CoBC** [manuelcortez/TWBlue#335]
To make it easier to find your way around in a private message conversation, I think that changing the presentation of private messages would be useful.
My idea is as follows:
- Delete the current "direct messages" and "send direct messages" buffers
- Create a unique "Messages" buffer
- In this buffer, insert sub-buffers, one per conversation, for example "conversation with x", "conversation with y", "conversation with z"
- In these sub-buffers, display the messages sent and received in the order in which they were received and sent, in order to ensure that a real conversation is displayed.
Maybe it could be made configurable?
Thank you.

**Comment by manuelcortez** [comment]
Hi there,
I could have done that, but this has many implications and up to this point this will reduce functionality instead of adding value to the client. These are the points why the buffers works in the way they do now:

* As opposed to other endpoints, all endpoints related to direct messages are heavily limited. Max of 50 items per call, when others allow 200, and even less calls every 15 minutes. so we are dramatically limited about how many items we will retrieve every time.
* There are no methods to retrieve or even identify conversations. Not directly, at least.

In conclusion, this kind of reorganisation in conversations will be relatively difficult to implement, having that we will retrieve all items in a single API call, then redistribute all items received into buffers, when normally every buffer retrieves items directly with its own API call, and we will not have any benefit regarding presentation and more buffers to create and handle.
